heft-plugins/heft-api-extractor-plugin/package.json

```diff
@@ -9,6 +9,21 @@
   },
   "homepage": "https://rushstack.io/pages/heft/overview/",
   "license": "MIT",
+  "exports": {
+    "./lib/*": {
```

Comment:
Note that we may need a `"./lib/*.json": "./lib-esm/*.json"` rule in most package.json exports